### DISCUSSION TITLE
Pagina vacunacion

### DISCUSSION THREAD

**Discussion by saraivetesigv** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#321] [Q&A]
Alguien que me pueda ayudar a saber como obtener los iconos

**Comment by Daniel-Valente** [comment]
Hola, yo los iconos los estoy usando de la página https://www.flaticon.es/ no encontré los mismo, pero si algunos similares

**Reply by saraivetesigv** [reply]
gracias, voy a ver cuales le quedan

**Comment by LeonardoRosalesG** [comment]
Yo uso esta app para buscar y usar iconos.:
https://www.microsoft.com/es-mx/p/pichon-iconos-gratis/9nk8t1kshffr?activetab=pivot:overviewtab

Cuidado con usarlos con fines de lucro, necesitas licencia para eso.

**Reply by saraivetesigv** [reply]
gracias, solo necesito para la pagina

**Comment by OmarVenturaP** [comment]
También Font Awsome tiene varios iconos y gratuitos. Yo allí encontré todos los de mi página. Puedes darte una vuelta por allí.

**Reply by saraivetesigv** [reply]
gracias

**Comment by doguedogue** [comment]
iconos8

https://iconos8.es/icons/set/help

**Reply by saraivetesigv** [reply]
gracias

**Comment by DanyVeneno** [comment]
Hola Sarai 
1-en tu buscador escribes cdn cloudflare fontawesome te va llevar a u  sitio
2-copias el primer link que te aparece version 6.0
3-lo pegas en le head de tu pagina donde lo vas a usar dentro de una etiqueta <link>
4-regresas a tu buscador y escribes font awesome te va allecar a un sitio
5-buscas el linl de Icons y le das click
6- buscas free icons le das click y te aparecen todos los iconos que puedes usar gratis
7-en el buscador ahi mismo dentro del sitio escribes el tipo de icono que quieres buscar y te aparece o solo dandole scroll vas viendo los que hay
8-bueno te pongo el link https://fontawesome.com/search?m=free
9-https://cdnjs.com/libraries/font-awesome
10-copias la etiqueta <i> dentro de tu documento html en la parte donde la quieras usaar y para editarla en tu documento css te refieres a ella con su classe.
Espero te sirva buena tarde.
![Screenshot_1](https://user-images.githubusercontent.com/89219507/157531921-c954f69a-39fd-48a3-bdc8-c87548b359f2.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/89219507/157531928-233b847b-a9a9-44d8-8ec9-2d9c730e7a53.jpg)
![Screenshot_3](https://user-images.githubusercontent.com/89219507/157531930-6cb3a6f4-e2d7-4380-9d8b-099360f18d58.jpg)
![Screenshot_4](https://user-images.githubusercontent.com/89219507/157531932-51381fd8-bd15-4bd3-b992-0728277c5f29.jpg)

**Reply by saraivetesigv** [reply]
muchas gracias, por la explicació, me sirvio mucho.

**Reply by JoseCruzR** [reply]
Me salvaste la vida hermano, asi como explicas con peras y manzanas es lo que necesito, me cuesta mucho esto y lo explicaste genial, muchisimas gracias.

**Comment by DanyVeneno** [comment]
Bien de eso se trata Saludos

**Comment by DanyVeneno** [comment]
De nada José Cruz si lo que preguntan me los se no duden en que les contestare buena noche
